app/utils/validators/forms/FieldValidator.scala

```diff
@@ -122,4 +113,16 @@ object FieldValidator {
   val areAllElementsUnique: Iterable[_] => Boolean = (input: Iterable[_]) => input.toSet.size == input.size
 
   val ofPattern: String => String => Boolean = (pattern: String) => (input: String) => input.matches(pattern)
+
+  val validDucr: String => Boolean = (input: String) => input.matches("""[0-9][A-Z][A-Z][0-9A-Z\(\)\-/]{6,32}""")
+
+  val validDucrIgnoreCase: String => Boolean = (input: String) => validDucr(input.toUpperCase)
```

Comment:
Ah, I just realised that with previous ticket I should fix the DUCR validation in all services not only in one...

Reply:
Hmm, can you use this validDucr method?

```
private val ducrPattern = Pattern.compile("[0-9]{1}[A-Za-z]{2}[0-9]{12}[-]{1}[-/()A-Za-z0-9]{1,19}")
val isValidDucr: String => Boolean = ducrPattern.matcher(_).matches()
```
This is from https://github.com/hmrc/customs-declare-exports-frontend/pull/1025/files#diff-787279139998872c746e28cb5b142d9bR169

Reply:
Or you can leave it, I can do this after you merge your PR, because this can break some tests, let's not combine thing together :)